docs/modules/ROOT/pages/tasks/build.adoc

```diff
@@ -0,0 +1,133 @@
+
+[[show-sbom-0.1]]show-sbom (0.1):: Shows the Software Bill of Materials (SBOM) generated for the built image in CyloneDX JSON format. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/show-sbom/0.1/README.md[README.md]
+
+[[show-sbom-rhdh-0.1]]show-sbom-rhdh (0.1):: Shows the Software Bill of Materials (SBOM) generated for the built image in CyloneDX JSON format. The 'task.*' annotations are processed by Red Hat Developer Hub (RHDH) so that the log content can be rendered in its UI. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/show-sbom-rhdh/0.1/README.md[README.md]
+
+[[slack-webhook-notification-0.1]]slack-webhook-notification (0.1):: Sends message to slack using incoming webhook See also: https://github.com/konflux-ci/build-definitions/tree/main/task/slack-webhook-notification/0.1/README.md[README.md]
+
+[[source-build-0.1]]source-build (0.1):: Source image build. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/source-build/0.1/README.md[README.md]
+
+[[source-build-oci-ta-0.1]]source-build-oci-ta (0.1):: Source image build. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/source-build-oci-ta/0.1/README.md[README.md]
+
+[[summary-0.1]]summary (0.1):: Summary Pipeline Task. Prints PipelineRun information, removes image repository secret used by the PipelineRun. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/summary/0.1/README.md[README.md]
+
+[[summary-0.2]]summary (0.2):: Summary Pipeline Task. Prints PipelineRun information, removes image repository secret used by the PipelineRun. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/summary/0.2/README.md[README.md] https://github.com/konflux-ci/build-definitions/tree/main/task/summary/0.2/MIGRATION.md[MIGRATION.md]
+
+[[tkn-bundle-0.1]]tkn-bundle (0.1):: Creates and pushes a Tekton bundle containing the specified Tekton YAML files. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/tkn-bundle/0.1/README.md[README.md]
+
+[[update-deployment-0.1]]update-deployment (0.1):: Task to update deployment with newly built image in gitops repository. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/update-deployment/0.1/README.md[README.md]
+
+[[update-infra-deployments-0.1]]update-infra-deployments (0.1):: Clones redhat-appstudio/infra-deployments repository, runs script in 'SCRIPT' parameter, generates pull-request for redhat-appstudio/infra-deployments repository. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/update-infra-deployments/0.1/README.md[README.md]
+
+[[upload-sbom-to-trustification-0.1]]upload-sbom-to-trustification (0.1):: Upload an SBOM file to [Trustification] using the [BOMbastic] API. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/upload-sbom-to-trustification/0.1/README.md[README.md]
```

Comment:
The `[Trustification]` and `[BOMbastic]` references are hyperlinks in the README but are not rendered properly here.